odoo/addons/base/data/res_partner_demo.xml

```diff
@@ -104,16 +104,16 @@
         </record>
 
         <record id="res_partner_5" model="res.partner">
-            <field name="city">Florenville</field>
-            <field name="country_id" ref="base.be"/>
+            <field name="city">Uuearu</field>
+            <field name="country_id" ref="base.ee"/>
```

Comment:
The purpose of this partner was to be a french speaking one https://github.com/odoo/odoo/commit/65df9635a80bbd76026a960cc19ddbdf1e0e69b0